docs/device-settings-ping360.md

```diff
@@ -0,0 +1,17 @@
+![Ping360 Settings](/ping-viewer/images/viewer/ping360-settings.png)
+
+## Menu Items
+
+- **Range:** Range control slider, performs the necessary sensor configuration for such distance range.
+- **Receiver Gain:** Adjust sensor sensitivity, between **low**, **medium** and **high**.
+- **Sector Angle:** Changes the scan angle range, helps to increase the scan time in specific areas.
+- **Reset Settings:** Reset sensor configuration, the default values for sample and scan will be used after that.
```

Comment:
what are `sample` and `scan` here?

Reply:
"Reset the settings in this panel to their default configuration"